### ISSUE TITLE
Gray text that is italicized is reported to be too hard to read

### ISSUE THREAD

**Issue by lbrannan-hub** [religious-ecologies/website#27]
ANDI reported that the color ratio fails for certain italicized text, like this example below. I can personally read it, but it might be an issue down the line. (Using Safari on MacBook Air).
<img width="1397" alt="Screen Shot 2020-10-29 at 12 48 04 PM" src="https://user-images.githubusercontent.com/60223963/97605599-373ecd80-19e5-11eb-91cf-e2a786682385.png">
